packages/bpk-component-map/readme.md

```diff
@@ -12,56 +12,53 @@ npm install bpk-component-map --save-dev
 
 ```js
 import React from 'react';
-import BpkMap from 'bpk-component-map';
+import BpkText from 'bpk-component-text';
+import BpkMap, { BpkOverlayView } from 'bpk-component-map';
 
 export default () => (
   <BpkMap
-    containerElement={<div style={{ height: '400px' }} />}
-    mapElement={<div style={{ height: `100%` }} />}
-    defaultZoom={15}
-    defaultCenter={{
-      lat: 27.9881,
-      lng: 86.925,
+    zoom={15}
+    showControls={false}
+    panEnabled={false}
+    center={{
+      latitude: 27.9881,
+      longitude: 86.925,
     }}
-    options={{
-      zoomControl: false,
-      dragEnabled: false,
-    }}
-  />
+  >
+    <BpkOverlayView position={{ latitude: 27.9881, longitude: 86.925 }}>
+      <BpkText>Shibuya Crossing</BpkText>
+    </BpkOverlayView>
+  <BpkMap>
 );
 ```
 
 ## Accompanying HOCs
 
-### withScriptjs
+### withGoogleMapsScript
 
-`withScriptjs` is a HOC that loads the Google Maps Javascript, then loads the map. This is useful for when you don't already have the Google Maps Javascript loaded.
+`withGoogleMapsScript` is a HOC that loads the Google Maps Javascript, then loads the map. This is useful for when you don't already have the Google Maps Javascript loaded.
 
 If you intend to include multiple maps on one page, it's better to load the Google Maps Javascript elsewhere and not use this HOC, as it downloads the script every time it's used.
 
 ```js
 import React from 'react';
-import BpkMap, { withScriptjs } from 'bpk-component-map';
+import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
```

Comment:
Why is this imported? It's not used AFAICT

Reply:
Good spot! I'll remove it.